### ISSUE TITLE
bootstrapping with LightGBM seeds

### ISSUE THREAD

**Issue by kamil-kaczmarek** [neptune-ml/open-solution-home-credit#141]
There are following options:
* `seed` or `random_seed` -> set LGBM seed globally
* `data_random_seed`, `feature_fraction_seed` -> train 5 or more times on the full training set, changing these seeds each time. Then take the mean or rank-mean of the predictions across each row.

More: https://www.kaggle.com/c/home-credit-default-risk/discussion/58332
